src/groupeddataframe/splitapplycombine.jl

```diff
@@ -962,8 +962,8 @@ function copyto_widen!(res::AbstractVector{T}, x::AbstractVector) where T
     return res
 end
 
-function groupreduce!(res::AbstractVector{U}, f, op, condf, adjust, checkempty::Bool,
-                      incol::AbstractVector{T}, gd::GroupedDataFrame) where {U,T}
+function groupreduce!(res, f, op, condf, adjust, checkempty::Bool,
```

Comment:
Just maybe leve `res::AbstractVector` for documentation reasons. I understand it should be OK, or this also crashes?

Reply:
done